### PR TITLE
Feature/360514 add flat icons

### PR DESCRIPTION
**Pull request by ms-Danny**

### CODE REVIEW

**Review by SteRed — DISMISSED** [review]

**Review by mattsskea — CHANGES_REQUESTED** [review]
You did a really good job, its hard to figure out which direction to go with a lot of this when we have designers with 1 opinion, senior devs with another, other devs (Me) with other opinions and then the implementation and cascading effects from changes and how they impact MSDS and prod.

I have given you as much feedback as I could, after looking over everything. I have tried be be thorough but- I could of missed something so I think Pawel and or Francesco should take a glimpse also. I will do one more check on Monday but I feel your pain <3
Too many requested changes with miscoordination and too little time :(

But my overall comment would relate to the updated svg values, I know Stephan wants the values to match the names and I know Martin wants the names to match the names they use in the design team.
I would suggest finding out if our team and the content editors have the time to update any old svg implementations to use the new values, that means we need to update code and CE need to update selected icons from selection factories. If we can get the time to do this in our current and next sprint then we can ignore my comments regarding values.
If we can't get the time to do that, we need to fix the values and svg names in the PR

File: components/icons/icons.list.js [hunk 1]
```diff
@@ -1,165 +1,640 @@
...
-  { name: 'Edit', value: '/Static/dist/msds-spritemap.svg#edit', previewValue: '#edit' },
-  { name: 'Elearning', value: '/Static/dist/msds-spritemap.svg#elearning', previewValue: '#elearning' },
-  { name: 'Expertise', value: '/Static/dist/msds-spritemap.svg#expertise', previewValue: '#expertise' },
-  { name: 'Extend', value: '/Static/dist/msds-spritemap.svg#extend', previewValue: '#extend' },
-  { name: 'Filter', value: '/Static/dist/msds-spritemap.svg#filter', previewValue: '#filter' },
-  { name: 'Grid', value: '/Static/dist/msds-spritemap.svg#grid', previewValue: '#grid' },
-  { name: 'Home', value: '/Static/dist/msds-spritemap.svg#home', previewValue: '#home' },
-  { name: 'Image', value: '/Static/dist/msds-spritemap.svg#image', previewValue: '#image' },
-  { name: 'Information', value: '/Static/dist/msds-spritemap.svg#information', previewValue: '#information' },
-  { name: 'IP Cameras', value: '/Static/dist/msds-spritemap.svg#ip-cameras', previewValue: '#ip-cameras' },
```
[the remark this answers is not in the document]
Comment (ms-Danny) [reply components/icons/icons.list.js:46]: All the icons are there in the branch and referenced in the file try look at the file and not just changes, you should find them there as well

File: components/icons/icons.list.js [hunk 2]
```diff
@@ -1,165 +1,640 @@
...
-  { name: 'Knowledgebase', value: '/Static/dist/msds-spritemap.svg#knowledgebase', previewValue: '#knowledgebase' },
-  { name: 'Like', value: '/Static/dist/msds-spritemap.svg#like', previewValue: '#like' },
-  { name: 'Link', value: '/Static/dist/msds-spritemap.svg#link', previewValue: '#link' },
-  {
-    name: 'List-horisontal',
-    value: '/Static/dist/msds-spritemap.svg#list-horisontal',
```
[the remark this answers is not in the document]
Comment (ms-Danny) [reply components/icons/icons.list.js:52]: Both the files and the references are still in the branch

File: components/icons/icons.list.js [hunk 3]
```diff
@@ -1,165 +1,640 @@
...
+    name: 'Social, Instagram',
     value: '/Static/dist/msds-spritemap.svg#social-instagram',
     previewValue: '#social-instagram'
   },
   {
-    name: 'Social Linkedin',
-    value: '/Static/dist/msds-spritemap.svg#linkedin',
-    previewValue: '#linkedin'
+    name: 'Social, LinkedIn',
+    value: '/Static/dist/msds-spritemap.svg#social-linkedin',
```
[the remark this answers is not in the document]
Comment (ms-Danny) [reply components/icons/icons.list.js:494]: i dont see used anywhere in main solution. searched for msds-spritemap.svg#linkedin

i only found this one <use xlink:href="/Static/Img/Icons/icons.svg#linkedin"></use>

File: components/icons/icons.list.js [hunk 4]
```diff
@@ -1,165 +1,640 @@
...
+    previewValue: '#trade-show'
+  },
+  {
+    name: 'Training',
+    value: '/Static/dist/msds-spritemap.svg#training',
+    previewValue: '#training'
+  },
+  {
+    name: 'Transaction',
+    value: '/Static/dist/msds-spritemap.svg#transaction',
```
[the remark this answers is not in the document]
Comment (ms-Danny) [reply components/icons/icons.list.js:574]: the only reference i found in main for this is Source\Workspaces\WEB\Main\Web_dev\Milestone.Web.Main\Milestone.Web.Main.Site\Static\dist\icons.list.js

and that will be automatically updated with NPM i asked pawel as well

**Review by mattsskea — DISMISSED** [review]

**Review by fcchiaro83 — APPROVED** [review]

**Pull request merged**